Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/describe.go

```diff
@@ -602,6 +603,9 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 	return tabbedString(func(out io.Writer) error {
 		fmt.Fprintf(out, "Name:\t%s\n", pod.Name)
 		fmt.Fprintf(out, "Namespace:\t%s\n", pod.Namespace)
+		if pod.Annotations[allocator.ValidatedSCCAnnotation] != "" {
```

Comment:
Sorry, one more thing you've run afoul of :)  

Our convention for empty string checking is `len(...) > 0` which is inconsistently enforced, but still the convention.

Also, SCC isn't very descriptive - can we call it "Security Policy"?